### ISSUE TITLE
[Python] Implement new Concept Exercise: basics

### ISSUE THREAD

**Issue by BethanyG** [exercism/v3#1999]
## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of programming in Python.

## Learning objectives

- Know what a variable (_name_)  is.
- Know how to name a variable.
- Know how to bind and re-bind a _name_ to a value (_assign or re-assign a value to a variable or assign a value to a constant_).
- Know what a `function` is and how  to define one via `def`.
- Know what the `return` keyword is, and how to use it to return a value from a function.
- Know how to call a function.
- Know how to declare an integer and a float.
- Know how to use mathematical operators `+` and `*` on integers and/or floats.
- Know how to define single- and multiline comments.
- Know how to define docstrings

## Out of scope

- built-in types
- built-in data structures
- Classes
- Methods and other function usages (_functions as arguments, nested functions_).
- Memory and performance characteristics.
- Default parameterrs
- Variadic parameters. (_*args and **kwargs_)

## Concepts

`basics`:  
- function definition & `return` keyword
- integers & floats, 
- mathematical operators `+` and `*`
- single and muliti-line comments.
- updating variable values 
- using functions
- variables, variable assignment, & variable naming

## Prerequisites

There are no prerequisites for this exercise

## Resources to refer to

[What is Python?](https://docs.python.org/3/faq/general.html#what-is-python)
[What is Python good for?](https://docs.python.org/3/faq/general.html#what-is-python-good-for)
[Numbers (_Python docs tutorial_)](https://docs.python.org/3/tutorial/introduction.html#numbers)
[Numeric Types](https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex)
[Defining Functions](https://docs.python.org/3/tutorial/controlflow.html#defining-functions)
[What is the difference between arguments and parameters in Python ?](https://docs.python.org/3/faq/programming.html#what-is-the-difference-between-arguments-and-parameters)

### Hints

See doc links above in `Resources to refer to`

### After

- Any concept exercise covering built-in data types or built-in functions.

## Representer

This exercise does not require any specific representation logic to be added to the representer.

## Analyzer

This exercise does not require any specific representation logic to be added to the analyzer..

## Implementing

Tests should be written using unittest.TestCase and the test file named lasanga_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
